docs/guides/_include/define-env.rst

```diff
@@ -1,26 +1,57 @@
-Define the environment
-----------------------
+Define disk variables 
+---------------------
 
 For convenience and to reduce the likelihood of errors, set environment variables that refer to the devices that
-will be configured during the setup. Verify your target disk devices with ``lsblk``.
+will be configured during the setup.
 
-First, define variables that refer to the disk and partition number that will hold boot files:
+For many users, it is most convenient to place boot files (*i.e.*, ZFSBootMenu and any loader responsible for
+launching it) on the the same disk that will hold the ZFS pool. However, some users may wish to dedicate an entire
+disk to the ZFS pool or create a multi-disk pool. A USB flash drive provides a convenient mechanism for storing the
+EFI boot partition (**ESP**). Fortunately, this alternative configuration is easily realized by simply defining a
```

Comment:
This section looks good, but the EFI referenes should be dropped because this block also appears in the syslinux/MBR guide.

```suggestion
disk to the ZFS pool or create a multi-disk pool. A USB flash drive provides a convenient mechanism for storing the
boot partition. Fortunately, this alternative configuration is easily realized by simply defining a
```